.gitignore

```
@@ -52,7 +52,14 @@ coverage.xml
 .hypothesis/
 .pytest_cache/
 cover/
+models/output/
+models/output/*
+models/output/*.csv
```

Why do you need all three?
```suggestion
models/output/
```

---

okay